sql-db/hibernate-reactive/src/main/java/io/quarkus/ts/reactive/IdGenerator.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.ts.reactive;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.hibernate.reactive.id.ReactiveIdentifierGenerator;
+import org.hibernate.reactive.session.ReactiveConnectionSupplier;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
+public class IdGenerator implements ReactiveIdentifierGenerator<Integer> {
+    private final AtomicInteger lastId = new AtomicInteger(4);
```

Comment:
We need to avoid the use of "magic numbers" in the code. Specify this number as constant for example. Probably you start the id generation from 4 because of /import.sql, this doesn't seem the great way to do it.

Reply:
Magic number removed, thank you!